charms/worker/k8s/charmcraft.yaml

```diff
@@ -161,6 +161,32 @@ config:
         CIDR to use for Kubernetes services. After deployment it is
         only possible to increase the size of the IP range. It is not possible to
         change or shrink the address range after deployment.
+    local_storage_enabled:
+      type: boolean
```

Comment:
let's establish a default value.  I *THINK* the request was it's enabled by default.  so please add a `default: true`

Reply:
Done